apps/src/templates/GameButtons.jsx

```diff
@@ -101,6 +106,7 @@ UnconnectedGameButtons.displayName = 'GameButtons';
 
 export default connect(state => ({
   hideRunButton: state.pageConstants.hideRunButton,
+  useAgent: state.pageConstants.useAgent,
```

Comment:
It might be nice to specify `useMinecraftAgent` here, given that this is a pretty global template and "agent" is a pretty generic term

Reply:
Good call, I'll update.